wicket-examples/src/main/java/org/apache/wicket/examples/upload/UploadApplication.java

```diff
@@ -55,9 +61,22 @@ protected void init()
 		// Ensure folder exists
 		uploadFolder.mkdirs();
 
+		uploadsFileManager = new FolderUploadsFileManager(uploadFolder);
+
+		mountResource("/uploads", FileUploadResourceReference.createNewInstance(uploadsFileManager, Bytes.megabytes(100)));
```

Comment:
100M is too big. We deploy the examples app at https://examples9x.wicket.apache.org and this may consume out disks quickly.